pkg/zrouter/zrouter.go

```diff
@@ -95,6 +101,16 @@ func New(metricsServer metrics.TaskMetrics, config *Config) ZRouter {
 		metricsServer: metricsServer,
 		config:        config,
 	}
+
+	if config.SystemMetrics.Enable {
+		if err := metrics.RegisterSystemMetrics(metricsServer); err != nil {
+			logger.GetLoggerFromContext(context.Background()).Errorf("Error registering metrics %v", err)
+		}
+
+		updateInterval := config.SystemMetrics.UpdateInterval
+		go metrics.UpdateSystemMetrics(metricsServer, updateInterval)
```

Comment:
I actually don't like not to have control over this go routine... to listen to errors, or anything that could happen. What do you think?